### PR TITLE
Code review TEST

### PR DESCRIPTION
**Pull request by yujin0131**
Code review TEST

### CODE REVIEW

File: src/main/java/com/bis/instagrarn/ProfileController.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.multipart.MultipartFile;
+
+import common.Common;
+
+/**
+ * Handles requests for the application home page.
+ */
+@Controller
+public class ProfileController {
```
Comment (yoonwooseong) [inline src/main/java/com/bis/instagrarn/ProfileController.java:23]: Good at separating sections

File: src/main/java/com/bis/instagrarn/ProfileController.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+				System.out.println("잘됨");
+			} catch (Exception e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+			
+		}
+		return Common.Profile.VIEW_PATH + "profile.jsp";
+	}
+}
```
Comment (yujin0131) [inline src/main/java/com/bis/instagrarn/ProfileController.java:67]: - [ ] fix more.

File: src/main/java/dao/UserDAO.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import vo.UserVO;
+
+public class UserDAO {
+	
+	@Autowired
+	DataSource dataSource;
+	
+	public int insert( UserVO vo ) {
+		JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
```
Comment (yoonwooseong) [inline src/main/java/dao/UserDAO.java:22]: It can be implemented more easily using 'Mybatis framework' as we know it.

Comment (yoonwooseong) [reply src/main/java/dao/UserDAO.java:22]: jdbc -> mybatis

File: src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml [hunk 4]
```diff
@@ -1,28 +1,48 @@
...
+			value="jdbc:mysql://13.209.7.121:3306/InstagrarnDB?useUnicode=true&amp;characterEncoding=UTF-8&amp;serverTimezone=Asia/Seoul&amp;useSSL=false" />
+		<beans:property name="username" value="InstagrarnMysql" />
+		<beans:property name="password" value="1111" />
+	</beans:bean>
+	
+	<beans:bean id="mulitpartResolver" class="org.springframework.web.multipart.commons.CommonsMultipartResolver">
+        <beans:property name="maxUploadSize" value="104857600"/>
+    </beans:bean>
+
+	<beans:bean class="com.bis.instagrarn.HomeController"/>
```
Comment (yoonwooseong) [inline src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml:42]: Not sure to do this

Comment (yoonwooseong) [reply src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml:42]: We need to study the beans object

File: src/main/java/com/bis/instagrarn/ProfileController.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+public class ProfileController {
+
+	@RequestMapping(value = "/addpost", method = RequestMethod.GET)
+	public String home() {
+
+		return Common.Profile.VIEW_PATH + "addpost.jsp";
+	}
+	
+	@RequestMapping(value = "/upload", method = RequestMethod.POST)
+	public String main(MultipartFile file, String content, String area, Model model) {
```
Comment (yoonwooseong) [inline src/main/java/com/bis/instagrarn/ProfileController.java:32]: It is also possible to create and implement boardVO :)

**Review by yoonwooseong — COMMENTED** [review]
Find what is lacking

File: src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml (hunk 4 above)
Comment (yujin0131) [reply src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml:42]: roger

File: src/main/java/com/bis/instagrarn/ProfileController.java (hunk 5 above)
Comment (yujin0131) [reply src/main/java/com/bis/instagrarn/ProfileController.java:32]: okay,,,

**Pull request merged**